gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -243,6 +251,9 @@ class {{ service.async_client_name }}:
 
         )
 
+        if CLIENT_LOGGING_SUPPORTED:
+            _LOGGER.debug(" {{ service.async_client_name }} client is successfully constructed.")
```

Comment:
This log also appears when running async code. Is that expected?

```
DEBUG    google.showcase_v1beta1.services.echo.client:client.py:605  EchoClient client is successfully constructed.
DEBUG    google.showcase_v1beta1.services.echo.async_client:async_client.py:258  EchoAsyncClient client is successfully constructed.
```

Reply:
The ruby logs have the package name in the logs, whereas we have the module name. Is there a benefit to including the PyPI package name as well?

```
google-cloud-secret_manager-v1: Created client for google.cloud.secretmanager.v1.SecretManagerService 
```

Reply:
> The log for the sync client appears when running async code. Is that expected?

This was fixed in https://github.com/googleapis/gapic-generator-python/pull/2265/commits/6f2bbee80037d39c0ce8fefe0b59b3b14875cd8f .

Disregard my comment about the package name. The module name provide sufficient information